partcad/src/partcad/context.py

```
@@ -733,6 +747,7 @@ async def supplier_carts_to_quotes(
         return quotes
 
     def get_provider(self, part_spec, params=None):
+        pc_logging.debug("Getting provider for %s" % part_spec)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)